docs/DeveloperGuide.md

```diff
@@ -2,7 +2,7 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
```

Comment:
Media class is shown as abstract in the UML diagram. However, it is not implemented as an abstract class. Is it supposed to be just shown as 'normal' class in the UML diagram?